drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/editor/menu/CellContextMenuViewImpl.java

```diff
@@ -97,24 +97,40 @@ public void enableDeleteCellMenuItem( final boolean enabled ) {
     @SuppressWarnings("unused")
     @EventHandler("cellContextMenuCut")
     public void onClickCellContextMenuCut( final ClickEvent e ) {
+        if ( isDisabled( cellContextMenuCut ) ) {
```

Comment:
@paulovmr @ederign @Rikkola @porcelli FYI (I got burned) "disabled" Bootstrap 3 items aren't "disabled" to DOM events... so you need to check whether they're actually disabled...